omod/src/main/java/org/openmrs/module/rwandahivflowsheet/web/model/HivFlowsheetFormData.java

```diff
@@ -217,7 +217,7 @@ public Date getArvStartDate() {
 	    			dorListOnEndDate = new LinkedHashSet<DrugOrder>();
 	    		//now check all orders to see if they're active on that date also, and add if so:
 	    		for (DrugOrder dorInner : rawOrders){
-	    			if (dorInner.isCurrent(endDate) && !dorListOnStartDate.contains(dorInner)) //only add each DrugOrder once.
+	    			if (dorInner.isActive(endDate) && !dorListOnStartDate.contains(dorInner)) //only add each DrugOrder once.
```

Comment:
Just need to confirm that this doesn't need to handle orders scheduled for the future.